apps/pie-microsite/src/_11ty/shortcodes/tokensTable/tokenTypes/elevation.js

```diff
@@ -0,0 +1,49 @@
+const { deindentHTML } = require('../../shortcode-utilities');
+const pieDesignTokenColours = require('../../../filters/pieDesignTokenColours');
+
+
+/**
+ * Builds the example elevation box-shadows to show on the token list item
+ * @param {string} token - the token value
+ * @param {*} tokenMetadata - the metadata for the token. data such as descriptions
+ * @param {path} path - the categorised token path (light or dark)
+ * @returns {string} - the elevation swatch example HTML string
+ */
+const buildElevationExample = (token, tokenMetadata, path) => {
+    const boxShadowValues = token.shadows.map(({
```

Comment:
I would move defining this variable down to right above where it is used, less cognitive load when reading the code that way :)